test/tla/cli-integration-tests.md

```diff
@@ -897,125 +897,93 @@ The outcome is: NoError
 EXITCODE: OK
 ```
 
-### check Rec1.tla succeeds
+### check Rec1.tla fails
 
 ```sh
 $ apalache-mc check --length=5 --inv=Inv Rec1.tla | sed 's/I@.*//'
 ...
-The outcome is: NoError
-...
-EXITCODE: OK
+EXITCODE: ERROR (255)
 ```
 
-### check Rec2.tla succeeds
+### check Rec2.tla fails
 
```

Comment:
can we demote these tests to `typecheck`, that is, the model checker fails, but the type checker does not.

Reply:
Sure, but rather than demote, I'd test for fail on `check`  _and_ success on `typecheck`

Reply:
yes, this is a good solution